src/master/flags.cpp

```diff
@@ -460,6 +460,18 @@ mesos::internal::master::Flags::Flags()
       "load an alternate allocator module using `--modules`.",
       DEFAULT_ALLOCATOR);
 
+  add(&Flags::hierarchical_recovery_factor,
+      "hierarchical_recovery_factor",
+      "Ratio of minimal re-registred agent before sending\n"
```

Comment:
"Ratio" usually implies two numbers (ratio of something to something else), doesn't it?..

Maybe something  like "Minimum fraction of known agents re-registered after leader election for the allocator to start generating offers"?

Reply:
You're right; I used your sentence it seems fine to me too.